docs/docsite/rst/playbook_guide/playbooks_loops.rst

```diff
@@ -256,6 +256,41 @@ When you run a task with ``until`` and register the result as a variable, the re
 
 If ``until`` is not specified, the task will retry until the task succeeds but at most ``retries`` times.
 
+.. _retry_with_timeout:
+
+Retrying a task with timeout
+----------------------------
+
+You can use the ``timeout`` keyword to set a time limit for the execution of the task code. 
+
+When you retry a task, ``timeout`` applies per each attempt and remains consistent -- it is not affected by the total time spent on a task.
```

Comment:
This hits on the relevant detail imo.

Reply:
My suggestion for this PR is to add this detail to the TASK_TIMEOUT config instead, and link to it in a note from this page.

Reply:
That would be a change here, right? - https://github.com/ansible/ansible/blob/devel/lib/ansible/config/base.yml#L1920
(aka different repo). And you're specifically talking about line 266 above?